server/jetty/src/main/java/io/deephaven/server/jetty/JettyBackedGrpcServer.java

```diff
@@ -85,20 +83,15 @@ public class JettyBackedGrpcServer implements GrpcServer {
     private static final String JS_PLUGINS_PATH_SPEC = "/" + JsPlugins.JS_PLUGINS + "/*";
 
     private final Server jetty;
-    private final JsPlugins jsPlugins;
     private final boolean websocketsEnabled;
 
     @Inject
     public JettyBackedGrpcServer(
             final JettyConfig config,
-            final GrpcFilter filter) {
+            final GrpcFilter filter,
+            final JsPlugins jsPlugins) {
```

Comment:
This adds a package private class to a public constructor, essentially rendering it non-public to anybody but us. Given the way we construct JettyBackedGrpcServer, it's probably reasonable to make this constructor package private. The other option is to make `JsPlugins` public.

I wish there were a way to force java compilation to fail in these cases?

Reply:
![image](https://github.com/deephaven/deephaven-core/assets/6764691/98ca30f3-a1da-4cec-91ea-40dde6a4b881)

Reply:
I bet https://errorprone.info/ can make this fail (or at least has an option for it).

Agreed, I can change this (as discussed briefly in slack) in either direction. I'd lean towards both being public, do you have a preferece?